RELEASE_INFO-6.7.md

```diff
@@ -107,6 +107,7 @@ Example usage:
 ## Administration
 
 ## Storefront
+- Fixed missing tooltips when hovering over product images in category listings. The `title` attribute is now added to the product name link which has the `stretched-link` class, allowing tooltips to appear when hovering over the image area.
```

Comment:
I don't think, we need an entry for this change in the release info. it is no feature and there are no breaks are relevant changes for other developers to be aware of